template/ios/Podfile

```diff
@@ -7,6 +7,8 @@ target 'HelloWorld' do
   config = use_native_modules!
 
   use_react_native!(:path => config[:reactNativePath])
+  # Or, to enable hermes on iOS, comment out the above line and uncomment the below:
+  # use_react_native!(:path => config[:reactNativePath], :hermes_enabled => true)
```

Comment:
What do you think about doing it like so?

```ruby
use_react_native(
  :path => config[:reactNativePath],
  # Or, to enable hermes on iOS, uncomment the below line:
  # :hermes_enabled => true
)
```

Reply:
I like that idea better! I just didn't think about it.